src/properties.cpp

```diff
@@ -170,21 +170,21 @@ void PropertiesTable::displaySimulation(QString id) {
 
     createNodeItem(rowCount, 0, "lattice");
     createPropertyItem(++rowCount, 1, "fill");
-    createValueItem(rowCount, 2, simConfig->lattice.fillName());
+    createValueItem(rowCount, 2, simConfig->lattices[0]->fillName());
```

Comment:
A different lattice node needs to be created per lattice object so that we can edit all of them in the side panel. See `displayLoadcase` for an example.